yarn.lock

```diff
@@ -110,10 +110,10 @@ __metadata:
   languageName: node
   linkType: hard
 
-"@babel/compat-data@npm:^7.12.13, @babel/compat-data@npm:^7.17.7, @babel/compat-data@npm:^7.20.5, @babel/compat-data@npm:^7.22.5":
-  version: 7.22.5
```

Comment:
There shouldn't be any changes to yarn.lock or the cache files. Can we revert those changes?